### PR TITLE
chore: pairing param fix and example

### PR DESCRIPTION
**Pull request by gpsanant**
I learned the hard way what the pairing parameters were yesterday :). So, I fixed them on the site and added an example.

### CODE REVIEW

File: docs/precompiled/0x08.mdx [hunk 1]
```diff
@@ -14,10 +14,10 @@ The input must always be a multiple of 6 32-byte values. 0 inputs is valid and r
 |-----------:|-----:|
 | `[0; 31]` (32 bytes) | x1 |
 | `[31; 63]` (32 bytes) | y1 |
-| `[64; 95]` (32 bytes) | x2 |
-| `[64; 95]` (32 bytes) | y2 |
 | `[64; 95]` (32 bytes) | x3 |
+| `[64; 95]` (32 bytes) | x2 |
```
Comment (peroket) [inline docs/precompiled/0x08.mdx:18]: I see we have a copy paste error here, could you update the byte ranges?

File: docs/precompiled/0x08.mdx [hunk 2]
```diff
@@ -26,3 +26,20 @@ The input must always be a multiple of 6 32-byte values. 0 inputs is valid and r
 | `[0; 31]` (32 bytes) | success | 1 if the pairing was a success, 0 otherwise |
 
 If the input is not valid, or if not enough gas was given, then there is no return data.
+
+## Example
+
+| Input | Output |
+|------:|-------:|
+| `0x2cf44499d5d27bb186308b7af7af02ac5bc9eeb6a3d147c186b21fb1b76e18da` | `1` |
```
Comment (peroket) [inline docs/precompiled/0x08.mdx:34]: Thanks a lot that's awesome :-)
Can I ask you to write an example in the playground and add it here?
For example, for precompiled contract 01 https://www.evm.codes/precompiled#0x01 we have the follwoing code
https://www.evm.codes/playground?unit=Wei&codeType=Mnemonic&code=%27jFirsNplace_parameters%20in%20memoryZ456e9aea5e197a1f1af7a3e85a3212fa4049a3ba34c2289b4c860fc0b0c64ef3whash~Y~28wvX2YZ9242685bf161793cc25603c231bc2f568eb630ea16aa137d2664ac8038825608wrX4YZ4f8ae3bd7535248d0bd448298cc2e2071e56992d0774dc340c368ae950852adawsX6YqqjDo_call~32JSizeX80JOffsetX8VSize~VOffset~1waddressW4QFFFFFFFFwgasqSTATICCALLqqjPut_resulNalonKon_stackqPOPX80qMLOAD%27~W1%20w%20jq%5Cnj%2F%2F%20_%20thKZW32QY0qMSTOREX~0xWqPUSHV0wargsQ%200xNt%20Ke%20Jwret%01JKNQVWXYZ_jqw~_

**Review by peroket — DISMISSED** [review]
Thanks for the example, that helps a lot! Would be awesome to fix some stuff too, but it's fine if you don't.

File: docs/precompiled/0x08.mdx (hunk 1 above)
Comment (gpsanant) [reply docs/precompiled/0x08.mdx]: oops! found a few errors on some of the other precompiles as well and pushed the index updates to my fork

File: docs/precompiled/0x08.mdx (hunk 2 above)
Comment (gpsanant) [reply docs/precompiled/0x08.mdx:34]: sure! would be fun to learn. It's super late locally but I'll get to it soon :)

File: docs/precompiled/0x08.mdx (hunk 1 above)
Comment (peroket) [reply docs/precompiled/0x08.mdx]: thanks a lot :-)

File: docs/precompiled/0x08.mdx (hunk 2 above)
Comment (gpsanant) [reply docs/precompiled/0x08.mdx:34]: oh, I can just write this in solidity, right?

Comment (peroket) [reply docs/precompiled/0x08.mdx:34]: We are more aimed at learning specific opcodes, but yes that would work too thanks :-)

Comment (gpsanant) [reply docs/precompiled/0x08.mdx:34]: added an example in solidity :)

**Review by peroket — APPROVED** [review]

File: docs/precompiled/0x08.mdx (hunk 2 above)
Comment (peroket) [reply docs/precompiled/0x08.mdx:34]: Thanks looks good

**Pull request merged**